src/keri/vc/protocoling.py

```diff
@@ -405,3 +405,221 @@ def presentationExchangeExn(hab, reger, said):
     del ims[:exn.size]
 
     return exn, ims
+
+
+class IpexHandler(doing.Doer):
+    """ Processor of `exn` IPEX messages.
+
+    """
+
+    resource = "/ipex"
+    persist = True
+
+    def __init__(self, hby, rgy, notifier, **kwa):
+        """ Initialize instance
+
+        Parameters:
+            hab (Habitat): local identifier environment
+            wallet (Wallet) credential wallet that will hold the credentials
+            ims (Optional(bytearray)): inbound message stream to process
+            notifier (Notifier): outbound notifications
+            **kwa (dict): keyword arguments passed to DoDoer
+
+        """
+        self.hby = hby
+        self.rgy = rgy
+        self.notifier = notifier
+        self.msgs = decking.Deck()
+        self.cues = decking.Deck()
+
+        super(IssueHandler, self).__init__(**kwa)
+
+    def recur(self, tyme):
+        if self.msgs:
+            msg = self.msgs.popleft()
+            serder = msg["serder"]
+            attrs = serder.ked["a"]
+
+            if not attrs["r"].startWith(self.resource):
```

Comment:
This check isn't necessary because messages won't end up here without the proper route, that't what the routing is for.  Think of it like an HTTP method handler, they don't need to check the path each time

Reply:
removed